### ISSUE TITLE
Cross platform

### ISSUE THREAD

**Issue by davelab6** [murasu/Hibizcus#3]
Like fontgoggles.org this is mac only app.

Resourcing/funding questions aside, what technical blockers and preferences do you see/have for upgrading it to be cross platform?

**Comment by murasu** [comment]
Only resourcing/funding. Nothing else.

**Comment by davelab6** [comment]
Ok no blockers is a good start :) 

Would you use pyqt, qt, web, druid?

**Comment by TreeBranches** [comment]
Agreed. Seems very unusual to make a tool like this exclusive to one OS.

Would happily donate to have this running on Windows.
